docs/contributing/7-contributing-icons.mdx

```diff
@@ -16,7 +16,7 @@ This page outlines the process of contributing an icon to the `@sumup/icons` pac
 3. Paste the SVG into your file and verify the code — refer to the "Caveats" section below.
 4. Commit the icons, they will automatically be optimized in the precommit hook using `svgo`.
 5. Add an icon object to the icons manifest file at [`packages/icons/manifest.json`](https://github.com/sumup-oss/circuit-ui/blob/9146e47a21dcd6880f437d1a47a0c54d5a164bfd/packages/icons/manifest.json). The icons are manually ordered alphabetically by icon category, then name (should match the file name), and finally by size (descending).
-6. Build the icons package (`yarn workspace @sumup/icons run build`) and run the Storybook (`yarn docs`). Verify that your icon renders correctly on the [Icons page](http://localhost:6006/?path=/docs/features-icons--page) (local link).
+6. Build the icons package (`npx lerna run build --scope=@sumup/icons`) and run the Storybook (`npm run docs`). Verify that your icon renders correctly on the [Icons page](http://localhost:6006/?path=/docs/features-icons--page) (local link).
```

Comment:
At some point we can look into finalizing the lerna-to-nx migration, too (we already have nx under the hood)